src/ReflectionEngine.php

```diff
@@ -281,4 +281,8 @@ public static function parseFileNamespace($fileName, $namespaceName)
         throw new ReflectionException("Namespace $namespaceName was not found in the file $fileName");
     }
 
+    public static function getLexer()
```

Comment:
Please add a return type declaration for this method

Reply:
Done